docs/src/performance.md

```diff
@@ -267,3 +267,13 @@ requires. It can thus be seen as a proxy for "energy used" and, as an extension,
     timing result, you need to set the analysis interval such that the
     `AnalysisCallback` is invoked at least once during the course of the simulation and
     discard the first PID value.
+
+## Performance issues due to false sharing
+False sharing is a known performance issue for systems with distributed caches. It also occurred
```

Comment:
```suggestion
[False sharing](https://en.wikipedia.org/wiki/False_sharing)
is a known performance issue for systems with distributed caches. It also occurred
```

Reply:
Thanks for the suggestions :raised_hands: